default.xml

```diff
@@ -29,6 +29,7 @@
   <project remote="github" name="apple/swift-format" path="swift-format" />
   <project remote="github" name="apple/swift-foundation" path="swift-foundation" />
   <project remote="github" name="apple/swift-installer-scripts" path="swift-installer-scripts" />
+  <project remote="github" name="apple/swift-inspect" path="swift-inspect" />
```

Comment:
This seems wrong and will break the checkout - there is no such repository. swift-inspect is a tool which resides in apple/swift.

Reply:
Updated